client/landing/stepper/declarative-flow/trial-wooexpress-flow.ts

```diff
@@ -110,6 +111,7 @@ const wooexpress: Flow = {
 
 		function submit( providedDependencies: ProvidedDependencies = {}, ...params: string[] ) {
 			recordSubmitStep( providedDependencies, intent, flowName, currentStep );
+			recordGTMDatalayerEvent( currentStep );
```

Comment:
I'm not too familiar with Stepper. But I see that there's a `useSideEffect` hook (optional) that we can use, ref the stepper readme file. It says it's for tracking functions and other side-effects.

@Automattic/martech-decepticons who's been working with Stepper. Should we use this, or can we keep the tracking logic in the submit function?